### PR TITLE
HCAP-1116 | Add CLI logic to invoke service

### PR DESCRIPTION
**Pull request by fwpushan**

### CODE REVIEW

File: server/scripts/services/change-cohort.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  }
+  console.log(`Found participant with ${participantId}`);
+
+  try {
+    await changeCohortParticipant({
+      cohortId: oldCohortDbObj.id,
+      participantId,
+      newCohortId: newCohortDbObj.id,
+      meta: {
+        user: options.user || 'system',
```
Comment (sunghwan-park-fw) [inline server/scripts/services/change-cohort.js:48]: How about adding user option to the usage?

File: server/scripts/services/change-cohort.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      },
+    });
+    return { success: true, message: 'Success', usageMessage: '' };
+  } catch (error) {
+    return { success: false, message: `Error: ${error}`, usageMessage: '' };
+  }
+};
+
+module.exports = {
+  changeCohort: serviceMethod,
```
Comment (sunghwan-park-fw) [inline server/scripts/services/change-cohort.js:59]: why do we need a different name?

**Review by sunghwan-park-fw — APPROVED** [review]

File: server/scripts/services/change-cohort.js (hunk 2 above)
Comment (fwpushan) [reply server/scripts/services/change-cohort.js:59]: no need I update it in next PR

File: server/scripts/services/change-cohort.js (hunk 1 above)
Comment (fwpushan) [reply server/scripts/services/change-cohort.js:48]: this is optional but I will add

**Pull request merged**